network/p2p/scoring/score_option.go

```diff
@@ -104,6 +105,43 @@ const (
 	// When a counter hits the DecayToZero threshold, it means that the peer did not exhibit the behavior
 	// for a long time, and we can reset the counter.
 	defaultDecayToZero = 0.01
+
+	// defaultTopicSkipAtomicValidation is the default value for the skip atomic validation flag for topics.
+	// We set it to true, which means gossipsub parameter validation will not fail if we leave some of the
+	// topic parameters at their default values, i.e., zero. This decision is since we are not setting all
+	// topic parameters at the current implementation.
+	defaultTopicSkipAtomicValidation = true
+
+	// defaultTopicInvalidMessageDeliveriesWeight this value is applied to the square of the number of invalid message deliveries on a topic.
+	// It is used to penalize peers that send invalid messages. By an invalid message, we mean a message that is not signed by the
+	// publisher, or a message that is not signed by the peer that sent it. We set it to -1.0, which means that with around 14 invalid
+	// message deliveries within a gossipsub heartbeat interval, the peer will be disconnected.
+	// The supporting math is as follows:
+	// - each staked (i.e., authorized) peer is rewarded by the fixed reward of 100 (i.e., DefaultStakedIdentityReward).
+	// - x invalid message deliveries will result in a penalty of x^2 * DefaultTopicInvalidMessageDeliveriesWeight, i.e., -x^2.
+	// - the peer will be disconnected when its penalty reaches -100 (i.e., MaxAppSpecificPenalty).
+	// - so, the maximum number of invalid message deliveries that a peer can have before being disconnected is sqrt(200/DefaultTopicInvalidMessageDeliveriesWeight) ~ 14.
```

Comment:
This is great info to know about the scoring. Can it be added to `network/p2p/scoring/README.md`?

Reply:
https://github.com/onflow/flow-go/pull/4492/commits/e6454d947f4c8dee0fb832c2eff084cf7550a15f